orc8r/lib/go/service/config/service_config.go

```diff
@@ -193,9 +193,13 @@ func getServiceConfigFilePath(moduleName, serviceName, configDir, oldConfigDir s
 	if fi, nerr := os.Stat(configFileName); nerr != nil || fi.IsDir() {
 		old := configFileName
 		configFileName = filepath.Join(oldConfigDir, moduleName, fmt.Sprintf("%s.yml", serviceName))
-		if fi, err := os.Stat(configFileName); err != nil || fi.IsDir() {
-			glog.Warningf("Cannot find '%s': %v, or Legacy Service Registry Configuration: '%s': %v",
-				old, nerr, configFileName, err)
+		if fi2, err := os.Stat(configFileName); err != nil || fi2 == nil || fi2.IsDir() {
+			if nerr != nil {
+				glog.Warningf("Cannot find '%s: %v' ",
+					old, nerr)
+			}
+			glog.Warningf("Cannot find Legacy Service Registry Configuration: '%s': %v",
```

Comment:
That shouldn't be the case, nil error should not cause any issues, see: https://play.golang.org/p/Cg0bPOBFzyo

Reply:
yes, you are right